website/docs/docs/deploy/ci-jobs.md

```diff
@@ -91,10 +91,20 @@ The green checkmark means the dbt build and tests were successful. Clicking on t
 
 If you're experiencing any issues, review some of the common questions and answers below.
 
+<details>
+  <summary>Temporary schemas aren't dropping</summary>
+  <div>
+    <div>If your temporary schemas aren't dropping after a PR merges or closes, this typically indicates you have overriden the <code>generate_schema_name</code> macro and it isn't using <code>dbt_cloud_pr_</code> as the prefix. To resolve this, change your macro so that the temporary PR schema name contains the required prefix. For example: 
```

Comment:
```suggestion
    <div>If your temporary schemas aren't dropping after a PR merges or closes, it typically indicates you have overriden the <code>generate_schema_name</code> macro and it isn't using <code>dbt_cloud_pr_</code> as the prefix. To resolve this, change your macro so that the temporary PR schema name contains the required prefix. For example: 
```